### DISCUSSION TITLE
Font doesn't display correctly after installation

### DISCUSSION THREAD

**Discussion by constanedes** [kovidgoyal/kitty#5591] [Q&A] [answered]
Hello everyone, can someone help me with this problem?

When installing kitty on my system (Ubuntu 22.04.01) it looks like this
![kitty](https://user-images.githubusercontent.com/78028296/196022811-485e9161-0e2f-4007-a4ea-c687d78d431c.png)

I have also tried with the same version of Kubuntu and the same thing happens. I searched various places about this but found nothing.
![neofetch](https://user-images.githubusercontent.com/78028296/196022601-17757e4d-9687-4553-98f4-825610bc9e7e.png)

**Comment by kovidgoyal** [comment, marked as answer]
Install the official binaries, the ubuntu package is very outdated so you wont get support for it here. https://sw.kovidgoyal.net/kitty/binary/

And check what font is being used with --debug-font-fallback and if its not an appropriate one change it with font_family in kitty.conf

**Reply by constanedes** [reply]
Thanks man, I'm going to try that.
